config/test.exs

```diff
@@ -7,7 +7,7 @@ import Config
 # Run `mix help test` for more information.
 config :hello, Hello.Repo,
   username: "postgres",
-  password: "postgres",
+  password: "adMin#12",
```

Comment:
@keliumJU it is not a good practice to upload passwords to the repository. This could represent a security breach!

Reply:
I agree, I think a config file should be used, that was just because I needed to create the project.

Reply:
password remove